src/juicy-popover.html

```diff
@@ -163,23 +162,59 @@
 
         JuicyPopoverElementPrototype.reposition = function () {
             var expandable = this.shadowRoot.querySelector("#expandable");
+            var expandableSlot = null;
```

Comment:
Regarding naming, I would say, the one above is a `-Slot` this one is just `expandable` element, isn't it?

Reply:
Thanks!
Renamed in 57ad7e2b0db98bc74825e2ab49530c17639492bf .